code/cobre.py

```diff
@@ -15,8 +15,30 @@
 import argparse
 from pathlib import Path
 
-
-def process_data(csv_file_p, output_json_p):
+# Define metadata
+metadata = {
+    "participant_id": {
+        "original_field_name": "[none given]",
```

Comment:
For ukbb I used the original column name in the original spreadsheet. we can perhaps have something similar?

Reply:
They do not provide a column name for this field unfortunately

Reply:
Whaaaat? Okay maybe we can do "N/A" in this case